CellProfiler_pipelines/cputils.py

```diff
@@ -0,0 +1,68 @@
+"""
+This collection of functions runs CellProfiler on the NF1 data for each plate and will rename the .sqlite outputed as the
+the method used to process the data.
+"""
+
+import os
+import pathlib
+
+
+def rename_sqlite_file(sqlite_dir_path: pathlib.Path, method: str):
+    """Rename the .sqlite file to be {method}.sqlite as to differentiate between the files
+
+    Args:
+        sqlite_dir_path (pathlib.Path): path to CellProfiler_output directory
+        method (str): name of the method used that was run through CellProfiler (e.g., PyBaSiC IC and CellProfiler feature extraction, etc.)
+    """
+    try:
+        # CellProfiler requires a name to be set in to pipeline, so regardless of plate or method, all sqlite files are outputed as "NF1_data.sqlite"
+        sqlite_file_path = pathlib.Path(f"{sqlite_dir_path}/NF1_data.sqlite")
+
+        new_file_name = str(sqlite_file_path).replace(
+            sqlite_file_path.name, f"{method}.sqlite"
+        )
+
+        # change the file name in the directory
+        pathlib.Path(sqlite_file_path).rename(pathlib.Path(new_file_name))
+        print(f"The file is renamed to {pathlib.Path(new_file_name).name}!")
+
+    except FileNotFoundError as e:
```

Comment:
It might be better to `raise` an exception here instead (or in addition to) printing something out. This way the code won't keep running, unless that is your intention!

Reply:
I am not 100% familiar with raise versus except, so in this case I don't really have an intention. I just know that this method of except will raise an error if there is no file out of all the files named `NF1_data.sqlite` so that means there is a problem. I think this will work just fine for this case as to me if I change it, it wouldn't improve the time it take to run.

I will look into `raise` in the future though since it might be best to use that more than try/excepts so thank you for bringing it up!